src/Codeception/Coverage/SuiteSubscriber.php

```diff
@@ -26,31 +39,62 @@ abstract class SuiteSubscriber implements EventSubscriberInterface
         'work_dir'       => null,
         'cookie_domain'  => null,
     ];
-
+    /**
+     * @var array
+     */
     protected $settings = [];
+    /**
+     * @var array
+     */
     protected $filters = [];
+    /**
+     * @var array
+     */
     protected $modules = [];
-
+    /**
+     * @var CodeCoverage
```

Comment:
In some other classes this variable woud be typed as `@var CodeCoverage|null`

Reply:
```php
    // src/Codeception/Coverage/Filter.php

    /**
     * @var CodeCoverage|null
     */
    protected $phpCodeCoverage = null;
```
However, the difference is that in `Filter` the variable is initialized to `null`.

I will change it anyway. Thanks